modules/ROOT/pages/osv2-usage.adoc

```diff
@@ -1,27 +1,50 @@
-= View Usage Graphs
+= Billing and Rate Limiting
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 :keywords: osv2, os2, object store, store, usage
 
+Object Store v2 uses the number of:
 
-You can view a chart indicating how many API requests have been consumed.
+* *HTTP transactions* to calculate billing
++
+See <<How Billing is Calculated>>.
+* *API requests* to calculate transactions per section (TPS) for rate limiting
++
+See <<Access Object Store API Request Usage Charts>>.
+
+== How Billing is Calculated
 
-* Every API call (via the connector or API) to Object Store v2 counts as one API request.
+* Only HTTP transactions count towards your monthly billing usage and quota.
 +
-All requests to the broker API, such as creating and deleting objects, are counted against the quota.
-* Up to 100 KB of object-payload data can be transferred in a single API call,
-which bills as one request.
-* Each additional 100 KB of payload data is charged as an additional API request. 
-* Monthly billing usage information is delayed by 24 hours.
+Each transaction (such as create, modify, or delete) on a store or an item counts as one billable unit against the quota.
```

Comment:
"Each transaction ... counts as one billable unit".  Here do we mean transaction as a HTTP Transaction?  It might be confusing as we are saying HTTP transaction counts as 1 billable unit. May be clarify that 1 or more based on the following heuristics
  - Payload 
  - Number of keys

Reply:
fixed